compressor/tgz.go

```diff
@@ -30,7 +32,12 @@ func (ctx *Tgz) options() (opts []string) {
 	if helper.IsGnuTar {
 		opts = append(opts, "--ignore-failed-read")
 	}
-	opts = append(opts, "-zcf")
+	_, err := exec.LookPath("pigz")
+	if err == nil {
+		opts = append(opts, "-I", "pigz", "-cf")
```

Comment:
Is this argument works?

<img width="398" alt="image" src="https://user-images.githubusercontent.com/5518/204246166-fba5b082-f286-481d-8bc6-6e5664e19905.png">

Reply:
@huacnlee Works on Linux, let me test brew pigz later.

Reply:
```
$ /usr/bin/tar --use-compress-program /usr/local/bin/pigz -cf demo.tar.gz demo
$ tar tf demo.tar.gz
demo
```

Fixed.